Patch by Sergei Golovan fixes mirroring font in canvas postscript when its
size is specified in pixels (negative).

--- tk8.5-8.5.4.orig/unix/tkUnixRFont.c
+++ tk8.5-8.5.4/unix/tkUnixRFont.c
@@ -260,6 +260,7 @@
     ftFont = GetFont(fontPtr, 0);
     fontPtr->font.fid = XLoadFont(Tk_Display(tkwin), "fixed");
     GetTkFontAttributes(ftFont, &fontPtr->font.fa);
+    fontPtr->font.fa.size = TkFontGetPoints(tkwin, fontPtr->font.fa.size);
     GetTkFontMetrics(ftFont, &fontPtr->font.fm);
 
     return fontPtr;
@@ -517,6 +518,7 @@
 				/* Actual font used to render the character */
 
     GetTkFontAttributes(ftFont, faPtr);
+    faPtr->size = TkFontGetPoints(tkwin, faPtr->size);
     faPtr->underline = fontPtr->font.fa.underline;
     faPtr->overstrike = fontPtr->font.fa.overstrike;
 }
